netsuite/11.7/modules/ROOT/pages/netsuite-examples.adoc

```diff
@@ -15,6 +15,11 @@ Obtain the inactive Employee records that have last names beginning with a speci
 * <<customer-advanced-search,Customer Advanced Search Filtering by the Date and Array Fields>>
 +
 Obtain the customer records that were created after 2015 and that are in the LEAD stage.
++
+* <<add-file, Add a File>>
++
+Add a file using the *Add* operation.
```

Comment:
This example does not go under the lead-in to this bullet list because that involves searching. The lead-in is this:

The following examples show how to perform searches in Anypoint Connector for NetSuite (NetSuite Connector):